src/Interpreters/ExpressionActions.cpp

```diff
@@ -206,7 +206,10 @@ void ExpressionActions::linearizeActions()
     {
         const auto & cur = data[reverse_index[input]];
         auto pos = required_columns.size();
-        actions[cur.position].arguments.front().pos = pos;
+
+        /// Const column does not have arguments
+        if (!actions[cur.position].arguments.empty())
+            actions[cur.position].arguments.front().pos = pos;
```

Comment:
Hm. It means that we have constant column in inputs list. This should never happen.
I think it would be better to add check here that input is `INPUT`, and fix wrong optimization.
I will try.

Reply:
> It means that we have constant column in inputs list. This should never happen.

Ok, got it, thanks